Data Structures/HashMap.java

```diff
@@ -0,0 +1,153 @@
+import java.util.Scanner;
+public class HashMap {
+ static class Pair {
+        int key;
+        int value;
+        Pair next;
+        Pair(int key, int value) {
+            this.key = key;
+            this.value = value;
+            this.next = null;
+        }
+    }
+    static class hashMap {
+        int size;
+        Pair[] arr;
+
+        hashMap(int size) {
+            this.size = size;
+            arr = new Pair[size];
+            for (int i = 0; i < size; i++) {
+                arr[i] = null;
+            }
+        }
+
+        //  Function to insert a key value pair in the hash map
+        public int hashFunction(int key) {
```

Comment:
Can we use Better Hash-Function here or not?

Reply:
Yes, Sir, we can use a better hash function to generate a key.
by using random numbers, timestamps, or prime numbers.